modular_zubbers/code/modules/opposing_force/code/equipment/antagonist_powers.dm

```diff
@@ -2,11 +2,11 @@
 	category = OPFOR_EQUIPMENT_CATEGORY_OTHER
 
 // Traitor
-/datum/opposing_force_equipment/antagonist_powers/uplink
-	item_type = /obj/item/uplink
-	name = "Syndicate Uplink"
-	description = "An old-school Syndicate uplink without a password and an empty TC account. Perfect for the aspiring operatives."
-	admin_note = "Traitor uplink without telecrystals."
+/datum/opposing_force_equipment/antagonist_powers/uplink //BUBBERSTATION EDIT
```

Comment:
you are making a bubber edit in a bubber file do you really need the tag

Reply:
> you are making a bubber edit in a bubber file do you really need the tag

Force of habit, removed it